src/javascript/app/pages/cashier/deposit_withdraw.js

```diff
@@ -223,10 +224,25 @@ const DepositWithdraw = (() => {
     const onLoad = () => {
         getCashierType();
         BinarySocket.send({ cashier_password: 1 }).then((response) => {
-            if ('error' in response) {
+            if (!response.error && response.cashier_password === 1) {
+                showMessage('cashier_locked_message');
+            } else if ('error' in response) {
```

Comment:
By swapping this and the above `if`, it would be more readable. (Also `!response.error` can be removed):
```js
if ('error' in response) {
    showError('custom_error', response.error.message);
} else if (response.cashier_password === 1) {
    showMessage('cashier_locked_message');
} else {
    ...
```

Reply:
yes it does, thanks :)